backbone/src/main/java/org/researchstack/backbone/ui/step/layout/ConsentReviewSubstepListStepLayout.java

```diff
@@ -99,8 +114,29 @@ protected ConsentSignatureBody createConsentSignatureBody(StepResult stepResult)
         }
         body.birthdate = usersBirthday;
 
+        // If one doesn't exist yet then set it to a blank string, otherwise the consent sig will have null value issues
```

Comment:
So, if sharingScope is null, but body.scope is not null, we want to clear out the body.scope? 

Does this work because setting the sharing scope is currently the last step? Do we have to clear out body.scope to have correct behavior? Seems like maybe if you had partially gone through consent, body.scope could be used to pre-populate the sharing step or something. Not required, just trying to think through what this does

Reply:
Nice catch, this is a bug, it should be....

if (sharingScope == null) {
        if (body.scope == null) {
             sharingScope = "";
        }
} else {
   body.scope = sharingScope;
}